### PR TITLE
feat(medusa): Add get inventory item endpoint

### PR DESCRIPTION
**Pull request by pKorsholm**
**What**
- Add get-inventory-item endpoint

### CODE REVIEW

File: integration-tests/modules/__tests__/inventory/index.spec.ts [hunk 1]
```diff
@@ -3,6 +3,7 @@ import { IInventoryServiceNext, IStockLocationService } from "@medusajs/types"
 import { ContainerRegistrationKeys } from "@medusajs/utils"
 import { ModuleRegistrationName } from "@medusajs/modules-sdk"
 import { createAdminUser } from "../../../helpers/create-admin-user"
+import e from "express"
```
Comment (riqwan) [inline integration-tests/modules/__tests__/inventory/index.spec.ts]: q: intentional?

**Review by riqwan — APPROVED** [review]
lgtm!

**Review by olivermrbl — APPROVED** [review]
LGTM

**Review by sradevski — APPROVED** [review]
LGTM, just check Riqwan's comment

**Pull request merged**